### PR TITLE
[PE-4552] Added Alert CE skeleton

### PR DESCRIPTION
**Pull request by janibolkvadze**
--- First implementation ---

https://ctl.atlassian.net/browse/PE-4552

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://a0f63ce578df8da2f48670582908526a81f4d995-chi-pr.enterprise.ctl.io" target="_blank"> https://a0f63ce578df8da2f48670582908526a81f4d995-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/776/ for more details.

File: src/chi/skeletons/alert.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  background-size: 75rem;
+}
+
+chi-alert:not(.hydrated) {
+  content: '';
+  display: flex;
+  font-size: 0;
+  width: 50rem;
+
+  &::before {
```
Comment (mattnickles) [inline src/chi/skeletons/alert.scss]: Let's remove the before element and use a single 100% container. Can we reduce the delay between the skeleton disappearing and the component content appearing?

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/807/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/808/ for more details.

File: src/chi/skeletons/alert.scss (hunk 1 above)
Comment (janibolkvadze) [reply src/chi/skeletons/alert.scss]: I removed the before element.
Also adjusted height of cases when size="" is specified `chi-alert[size="sm"]` and `chi-alert[size='lg']`.
With the current approach we achieve animation and height of web components before CE is loaded (page doesn't jump), but I'm still trying to figure out how to reduce the gap between the animation and component content. I opened a question in Stencil slack Chat here: https://app.slack.com/client/T77MU2PJL/C78FYDJH3/thread/C78FYDJH3-1585141134.007100

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://c094191d18f7b749c09b9685ea266d82bf657ab9-chi-pr.enterprise.ctl.io" target="_blank"> https://c094191d18f7b749c09b9685ea266d82bf657ab9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/809/ for more details.

**Comment by janibolkvadze** [comment]
> @janibolkvadze The margin + 100% width is causing the element to render outside of its container

Done. Also adjusted width of `type='toast'` and height when `title=''` is present for all the sizes.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://1af1b3e12893fa510d7ea777f2f86abf4f07b2c0-chi-pr.enterprise.ctl.io" target="_blank"> https://1af1b3e12893fa510d7ea777f2f86abf4f07b2c0-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/818/ for more details.

**Comment by jllr** [comment]
Great improvement @janibolkvadze!

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-4sobm0, https://ops.elasticbox.com/services/instances/i-az277l, https://ops.elasticbox.com/services/instances/i-oo21ob

**Pull request merged**